ui/lib/kubernetes/addon/components/page/role/details.hbs

```diff
@@ -0,0 +1,72 @@
+<PageHeader as |p|>
+  <p.top>
+    <nav class="breadcrumb" aria-label="breadcrumbs" data-test-breadcrumbs>
```

Comment:
You might be able to use `<Page::Breadcrumbs @breadcrumbs={{this.breadcrumbs}} />` if you want! But the component would need `data-test-crumb` added

In the pki redesign, we started defining crumbs in the route, on the controller, like [this](https://github.com/hashicorp/vault/pull/18268/files#diff-765f6f6f744dd96bd219401050ad1f88a67a32fb3805829f678b3d6054e06340R6) - and then extending the index route to add child crumbs - curious what your thoughts are!

Reply:
Oooh yes I forgot that you made that component! Our feature branch must be behind main so let me do some merging and take a look!

Reply:
I like the pattern you all went with in pki. We will look at setting up something similar. I think since we have other views that have bread crumbs that were built prior to the `Page::Breadcrumbs` component I will make a ticket to update them all at once rather than handling this view separately.